lib/active_model/serializer/collection_serializer.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveModel
+  class Serializer
+    class CollectionSerializer
+      NoSerializerError = Class.new(StandardError)
```

Comment:
this seems like a weird place for this.
Do we make new error types like this in other places?
should there be a common place where the errors are all kept?